packages/block-library/src/tag-cloud/style.scss

```diff
@@ -18,4 +18,18 @@
 		margin-left: 5px;
 		text-decoration: none;
 	}
+
+	&.is-style-pill a {
+		border: 1px solid var(--wp--preset--color--foreground);
+		border-radius: 5px;
```

Comment:
Does this need to be hard-coded?

Reply:
We could make it a block attribute, but it wouldn't do anything unless this style was used.

Reply:
It should only show up if the style is being used. We can also not set it to anything and let themes customize it for now.

Reply:
Using a custom variable in theme.json?

Reply:
No, the theme can just target it with CSS directly if they want to